src/blk/BlockDevice.cc

```diff
@@ -103,6 +103,9 @@ void IOContext::release_running_aios()
 BlockDevice::block_device_t
 BlockDevice::detect_device_type(const std::string& path)
 {
+#if defined(HAVE_LIBAIO) || defined(HAVE_POSIXAIO)
+    return block_device_t::aio;
```

Comment:
wrong indent. also, please move this down after that last `#if` block in this function, so it takes the lowest priority.